pages/guides/developing/dstorage-playback.en-US.mdx

```diff
@@ -52,6 +52,8 @@ function App() {
 
 Now that our providers are set up, we can add a text input for an IPFS CID or an IPFS/Arweave HTTP gateway URL.
 
+Note: For IPFS HTTP gateway URLs, the Studio provider currently only supports “path style” URLs and does not support “subdomain style” URLs. The Studio provider will support both styles of URLs in a future update.
```

Comment:
nit: can you use the `<Callout>` tag for this to increase visual prominence?

Reply:
Fixed in [ae8328d](https://github.com/livepeer/docs/pull/346/commits/ae8328d1b6f74b171e78692f42ad8a616b07bb81)